CMakeLists.txt

```diff
@@ -610,6 +610,22 @@ if (GPERFTOOLS_FOUND)
   endif ()
 endif ()
 
+if (NOT ARROW_ROOT_DIR AND VAST_PREFIX)
+    set(ARROW_ROOT_DIR ${VAST_PREFIX})
+endif ()
+find_package(arrow QUIET)
```

Comment:
We should make this an opt-out dependency with a `VAST_NO_ARROW` option to disable it explicitly.

Reply:
Implemented as suggested.